site/content/en/_index.html

```diff
@@ -0,0 +1,50 @@
++++
+title = "Agent Sandbox"
+linkTitle = "Agent Sandbox"
+description = "Agent Sandbox is a cloud native controller for sandboxes"
++++
+
+{{< blocks/cover title="Agent Sandbox" image_anchor="top" height="full" color="black" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary me-3 mb-4" href="/docs/">
+		Read the docs <i class="fas fa-arrow-alt-circle-right ms-2"></i>
+	</a>
+	<a class="btn btn-lg btn-secondary me-3 mb-4" href="https://github.com/kubernetes-sigs/agent-sandbox">
+		GitHub <i class="fab fa-github ms-2 "></i>
+	</a>
+	<p class="lead mt-5">Secure Agent Execution Environment</p>
+	{{< blocks/link-down color="info" id="description" >}}
+</div>
+{{< /blocks/cover >}}
+
+<div id="description" ></div>
+
+{{% blocks/lead color="primary"%}}
+<h1  class="text-center">Agent Sandbox provides a secure, and isolated execution layer to safely deploy autonomous AI agents on Kubernetes that generate and run untrusted code at scale.</h1>
```

Comment:
Would you use similar text we have in https://github.com/kubernetes-sigs/agent-sandbox/blob/main/README.md?